src/App.tsx

```diff
@@ -1,10 +1,40 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import classNames from 'classnames';
+import { useState, useEffect } from 'react';
+import { getTodos } from './api/todos';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
+import { Error } from './types/Error';
+import { Filter } from './types/Filter';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { wait } from './utils/fetchClient';
+import { getVisibleTodos } from './utils/getVisibleTodos';
 
-const USER_ID = 0;
+const USER_ID = 11135;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<Error>(Error.none);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+
+  const visibleTodos = getVisibleTodos(todos, filter);
+
+  if (errorMessage) {
+    wait(3000).then(() => setErrorMessage(Error.none));
+  }
```

Comment:
the state variable can't be changed without `useEffect` hook. also try to move this logic to `ErrorNotification` component on mount